app/views/layouts/_i18n_script.html.haml

```diff
@@ -1,4 +1,15 @@
-%script
+:javascript
   I18n.default_locale = "#{I18n.default_locale}";
   I18n.locale = "#{I18n.locale}";
   I18n.fallbacks = true;
+
+  switch(I18n.locale) {
+    case "es":
+      moment.lang('es');
+      break;
+    case "fr":
+      moment.lang('fr');
```

Comment:
can't we do `moment.lang(I18n.locale);`? AFAIK we both (moment and OFN) use the standard acronym for languages. As it is, it would require adding a bunch of case statements more. I can think of Portugal at least cc @mllocs.